desktop/src/main/java/bisq/desktop/main/dao/economy/dashboard/BsqDashboardView.java

```diff
@@ -96,13 +102,13 @@
     private AreaChart bsqPriceChart;
     private XYChart.Series<Number, Number> seriesBSQPrice;
 
-    private TextField marketCapTextField, availableAmountTextField;
+    private TextField avgPrice90TextField, medianPrice90TextField, marketCapTextField, availableAmountTextField;
     private Label marketPriceLabel;
 
     private Coin availableAmount;
-
```

Comment:
please only change stuff that actually changes something. Makes it more cumbersome to review...